### PR TITLE
Add set_state to LSTM.

### PR DESCRIPTION
**Pull request by fukatani**
This PR is for adding set_state method to LSTM link #1370 .

### CODE REVIEW

**Comment by rezoo** [comment]
Thank you for sending PR!
LGTM 👍

File: chainer/links/connection/lstm.py [hunk 1]
```diff
@@ -1,5 +1,8 @@
+import numpy
```
Comment (rezoo) [inline chainer/links/connection/lstm.py]: Could you put ``import numpy`` after ``six`` like that?
```python
import six

import numpy

import chainer
```

**Comment by rezoo** [comment]
LGTM except for the above comment! 👍

**Comment by fukatani** [comment]
Thanks for your review!
I reflected your pointing out.

**Comment by rezoo** [comment]
Great! I'll merge this PR.

**Pull request merged**